{"name": "test_zero_volume_cases[1w-CES-DAI]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 429 == 200\n +  where 429 = <Response [429]>.status_code", "trace": "tests/special/test_edge_cases.py:16: in test_zero_volume_cases\n    assert response.status_code == 200\nE   assert 429 == 200\nE    +  where 429 = <Response [429]>.status_code"}, "parameters": [{"name": "timeframe", "value": "'1w'"}, {"name": "pair", "value": "'CES-DAI'"}], "start": 1758131940187, "stop": 1758131940440, "uuid": "5b63a0ab-d46b-47d3-977b-3b81169f2b70", "historyId": "b264a708afd2a4f5863572c9ccdb095e", "testCaseId": "f1061a3867aae255684da5f0a2754e42", "fullName": "tests.special.test_edge_cases#test_zero_volume_cases", "labels": [{"name": "tag", "value": "special"}, {"name": "parentSuite", "value": "tests.special"}, {"name": "suite", "value": "test_edge_cases"}, {"name": "host", "value": "826d662bff36"}, {"name": "thread", "value": "2128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.special.test_edge_cases"}], "titlePath": ["tests", "special", "test_edge_cases.py"]}